src/components/image/Image.tsx

```diff
@@ -9,14 +10,20 @@ interface ImageProps {
 }
 export const Image = ({ src, alt, className, lazy = true }: ImageProps) => {
   const [loaded, setLoaded] = useState(false);
+  const imageOnErrorHandler = (
```

Comment:
no reason for this function to be declared again and again, consider moving it out of the Image function

Reply:
fixed